invenio_records_resources/services/files/service.py

```diff
@@ -65,13 +65,27 @@ def _get_record(self, id_, identity, action, file_key=None):
 
         return record
 
+    def _check_record_deleted(self, record, identity, include_deleted):
+        """Checks if record is deleted."""
```

Comment:
Very minor: reading `check something` I would expect that the function returns True or False. What about this?
```suggestion
    def _ensure_record_not_deleted(self, record, identity, include_deleted):
        """Ensure that the record exists (not deleted) or raise."""
```